src/app/widgets/Visualizer/index.jsx

```diff
@@ -934,6 +935,22 @@ class VisualizerWidget extends PureComponent {
         }
     }
 
+    rotaryAxisFunctions = {
+        JOG_PLUS: (_, { axis }) => {
+            // TODO - delete this and write logic
+            console.log('A + ', axis.a);
+        },
+        JOG_MINUS: (_, { axis }) => {
+            // TODO - delete this and write logic
+            console.log('A - ', axis.a);
+        },
+        UPDATE_STATUS: (_, { command }) => {
+            // TODO - delete console log and fire this event
+            console.log('Controller command: ', command);
+            // controller.command(command, store.get('rotaryAxisStatus', false));
+        }
+    }
+
```

Comment:
Not necessary, like mentioned before the a axis jog shortcuts should be moved to where all the other jog shortcuts are, and use the same function they use to determine jogging